src/utils/mathFormulas.js

```diff
@@ -0,0 +1,7 @@
+const stdev = (array) => {
+  const n = array.length;
+  const mean = array.reduce((a, b) => a + b) / n;
```

Comment:
you could use `_.sum` instead of a reduce here

Reply:
changed to use `_.sum()`